app/views/users/index.html.erb

```diff
@@ -1,3 +1,6 @@
+<div class="logout-link">
+<%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+</div>
 <% @users.each do |user| %>
   <%= render partial: 'partials/user', locals: { user: user } %>
 <% end %>
```

Comment:
- [ ] There is an error raised here in the `user#index` view when I try to sign up a new user, Please fix that.

![image](https://github.com/Nessrine88/MyBlog2/assets/68796232/baf2938a-587b-4a80-b726-8eeff23b9e66)